apps/ledger-live-desktop/src/renderer/screens/manager/FirmwareUpdate/index.tsx

```diff
@@ -61,13 +57,13 @@ const FirmwareUpdate = (props: Props) => {
     error,
     onReset,
   } = props;
-  const { setDrawer } = useContext(context);
```

Comment:
Oh ! I guess you were blocked when you wanted to add a `cancel` drawer, while the update drawer was still opened ?